assets/scss/main.css.scss

```diff
@@ -32,3 +32,9 @@
 .reversefootnote {
   @include u-font-family("mono");
 }
+
+.usa-layout-docs__sidenav {
+  position: sticky;
+  top: 2rem;
+  height: 100vh;
```

Comment:
For setting the height we can use a utility class from uswds, `height-viewport`

https://designsystem.digital.gov/utilities/height-and-width/

Reply:
Updated to use the USWDS class